Makefile

```diff
@@ -14,7 +14,6 @@ DESIGN_DIR=design
 DESIGNS := $(shell find $(SOURCE_DIR)/$(DESIGN_DIR) -path $(SOURCE_DIR)/vendor -prune -o -name '*.go' -print)
 
 GOLINT_DIRS=$(shell go list -f {{.Dir}} ./... | grep -v -E "vendor|app|client|tool/cli")
```

Comment:
Okay, I think we should keep the pkgs how they are `design and test` should also be analyzed. Go report system analyzes them all + those pkgs aren't auto generated.

Regarding the location of the variable I'll move it